src/T2DMap.cpp

```diff
@@ -4611,13 +4611,42 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     if (e->modifiers() & Qt::ControlModifier) { // Increase rate 10-fold if control key down - it makes scrolling through
                                                 // a large number of items in a listwidget's contents easier AND this make it
                                                 // easier to zoom in and out on LARGE area maps
-        delta *= 10;
+        delta *= 5;
     }
     if (delta != 0) {
         mPick = false;
-        int oldZoom = xyzoom;
-        xyzoom = qMax(3, xyzoom + delta);
+        qreal oldZoom = xyzoom;
+        xyzoom = qMax(3.0, xyzoom * pow(1.07, delta));
+
         if (oldZoom != xyzoom) {
+            const float widgetWidth = width();
```

Comment:
Both `width()` and `height()` return `int`s - do you need `static_cast<float>(`...`)` to suppress type conversion warnings?

Reply:
There's no such warning even with gcc's `-Wall`.